src/email.rs

```diff
@@ -10,27 +9,27 @@ use std::fs;
 use std::io;
 use std::path::Path;
 
+pub struct EmailTemplate {
+    pub subject: &'static str,
+    pub body: &'static str,
+}
+
+pub fn get_template() -> EmailTemplate {
+    EmailTemplate {
+        subject: "[{{provider}}] {{location}} eSIM",
+        body: include_str!("../templates/email_template.html"),
+    }
+}
```

Comment:
This looks rather like a constructor. The `include_str!` doesn't actually load anything from disk since it's done at compile time. It might be more idiomatic to write this like
```suggestion
impl EmailTemplate {
    pub fn new() -> Self {
        Self {
            subject: "[{{provider}}] {{location}} eSIM",
            body: include_str!("../templates/email_template.html"),
        }
    }
}
```

Reply:
Great suggestion! I honestly forget about impl blocks more than I'd like. This has been applied.